### DISCUSSION TITLE
Difference between observed_feat_dynamic_real and feat_dynamic_real?

### DISCUSSION THREAD

**Discussion by GeoffNN** [SalesforceAIResearch/uni2ts#99] [Q&A]
Hi -- great work, thanks for releasing the paper and codebase!

I was digging through the code and noticed that the forward in the lightning model takes `feat_dynamic_real` and `observed_feat_dynamic_real`, but I couldn't find any reference to the `observed` variant in the examples. Same thing for `past_dynamic_real` and `past_observed_dynamic_real`.
Could you expand on the difference between the two?
Thanks

**Comment by gorold** [comment]
observed is a binary tensor, indicating whether the corresponding value is observed or not (1 for observed, 0 for unobserved i.e. missing value).
